pepper-apis/studybuilder-cli/studies/osteo/study-workflows.conf

```diff
@@ -602,16 +544,6 @@
             && !user.studies["CMI-OSTEO"].hasAgedUp()
           """
         },
-        {
-          "type": "ACTIVITY",
-          "activityCode": "FAMILY_HISTORY",
-          "expression": """user.studies["CMI-OSTEO"].forms["FAMILY_HISTORY"].isStatus("CREATED", "IN_PROGRESS")"""
-        },
-        {
-          "type": "ACTIVITY",
-          "activityCode": "FAMILY_HISTORY_SELF",
-          "expression": """user.studies["CMI-OSTEO"].forms["FAMILY_HISTORY_SELF"].isStatus("CREATED", "IN_PROGRESS")"""
-        },
```

Comment:
Removing this is right?

Reply:
If workflow from is "FAMILY_HISTORY" its removed even if TO is something else
Other is duplicate entry
Its good to me